MonoGame.Framework/Random.cs

```diff
@@ -0,0 +1,386 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Xna.Framework
+{
+    public static partial class MathHelper
+    {
+        /// <summary>
+        /// Class for generating random numbers consistently when using Mono or Microsoft compilers.
+        /// </summary>
+        public class Random
```

Comment:
I'm not in favour of nested classes in a public API.  I would rather see this in the `MonoGame.Utilities` namespace rather than nested in XNA's `MathHelper` class.